lib/opte/src/engine/print.rs

```diff
@@ -207,9 +207,9 @@ pub fn print_lft_flow(
         t,
         "{}\t{}\t{}\t{}\t{}\t{}\t{}",
         flow_id.proto.to_string(),
-        flow_id.src_ip.to_string(),
+        flow_id.src_ip().to_string(),
```

Comment:
I know these `to_string()` are pre-existing but I'm not sure the above comment makes sense about custom `Display` implementations. `to_string()` would still use it

Reply:
Yeah, this exact thing comes up in #480, wherre the compiler has realised this fact.

I've realised we accidentally lost the nicer `Protocol` in Display/print, so I'll fix that with the comments here.